helm/odigos/templates/instrumentor/clusterrole.yaml

```diff
@@ -25,6 +25,18 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
I think we only read from the configmap, right? can we keep here only `get`, `list` and `watch`?